tests/effective_dps_test.cpp

```diff
@@ -484,31 +484,31 @@ static void check_shortswords( const std::function<Approx( const std::string & )
                                &calc_expected_dps )
 {
     SECTION( "shortswords" ) { // expected value 22
-        CHECK( calc_expected_dps( "scimitar" ) == 22.0 );
+        CHECK( calc_expected_dps( "scimitar" ) == 24.3 );
         CHECK( calc_expected_dps( "butterfly_swords" ) == 22.0 );
         CHECK( calc_expected_dps( "cutlass" ) == 22.0 );
         CHECK( calc_expected_dps( "sword_bayonet" ) == 22.75 );
-        CHECK( calc_expected_dps( "kukri" ) == 22.75 );
+        CHECK( calc_expected_dps( "kukri" ) == 26.47 );
```

Comment:
This should probably drop, 4 points above the expected value is a lot.

Reply:
Understood, I'll pull the damage down a point or two.